### DISCUSSION TITLE
Migration from Standalone Linux to Windows-Based Docker

### DISCUSSION THREAD

**Discussion by MrDevoHat** [netbox-community/netbox#9840] [Q&A] [answered]
Hello,

I am working on migrating my docker instance from a Linux VM to a Windows-based Docker. The documentation is very clear for the export and import process for replicating the DB in a standalone system, but I am having issues restoring the exported SQL database in to the docker container. 

Netbox is running in docker seamlessly, but the commands in the wiki are generating an error.

With my backup.sql in the root directory of the project, and the netbox-specific containers stopped, I am running:

docker-compose exec -T postgres sh -c 'psql -U $POSTGRES_USER $POSTGRES_DB'

Which ends up hanging, and shows no changes on restart.

Does anyone have experience with a standalone to docker migration?

Thanks!

**Comment by ryanmerolle** [comment, marked as answer]
I do not know the T flag for exect, but if you are using netbox-docker, this [wiki page for netbox-docker](https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting) should help specifically the [Database Operations Section](https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting#database-operations).

I went through it recently for a migration to validate the material was up to date.
